charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -70,7 +70,7 @@ resources:
 
 priorityClassName: ""
 nodeSelector: {}
-tolerateAllTaints: true
```

Comment:
I do think we should change the default but unsure if it's "safe" by helm standards to do it in a point release.

wdyt @ayberk

Reply:
Hmm this should be mostly fine, but it _might_ still break the existing workflow for some customers with basically no heads-up.

I'd rather change the behavior with a release so we can at least have a note in the release notes.

Reply:
Agree, How about we keep the current behavior for now, and turn this flag off in the next release(which should be pretty soon)

Reply:
Updated